release/README.md

```diff
@@ -23,10 +23,15 @@ Please refer to the following table to view the detailed release information for
 
 The driver containers released support the following Intel GPUs:  
 - [Intel® Data Center GPU Flex 140](https://www.intel.com/content/www/us/en/products/sku/230020/intel-data-center-gpu-flex-140/specifications.html)   
--  [Intel® Data Center GPU Flex 170](https://www.intel.com/content/www/us/en/products/sku/230019/intel-data-center-gpu-flex-170/specifications.html)
+- [Intel® Data Center GPU Flex 170](https://www.intel.com/content/www/us/en/products/sku/230019/intel-data-center-gpu-flex-170/specifications.html)
+- [Intel® Data Center GPU Max Series](https://www.intel.com/content/www/us/en/products/details/discrete-gpus/data-center-gpu/max-series.html) support starting from Driver Version 2.2.0 for RHOCP 4.14.11 and above.  
```

Comment:
Can we add
notes: It is verified on Max1100 Card

Reply:
Done